src/systems/mod.rs

```diff
@@ -27,10 +28,10 @@ pub use self::{
     animation::AnimationSystem, autoblaster_system::AutoBlasterSystem, blast::BlastSystem,
     boss::BossSystem, collision_detection::CollisionDetectionSystem, consumable::ConsumableSystem,
     defense::DefenseSystem, enemy::EnemySystem, enemy_collision_handler::EnemyCollisionSystem,
-    enemy_hit::EnemyHitSystem, enemy_spawn::SpawnerSystem, gamemaster::GameMasterSystem,
-    hitbox_system::HitboxSystem, item::ItemSystem, manualblaster_system::ManualBlasterSystem,
-    planets::PlanetsSystem, player_hit::PlayerHitSystem, spaceship::SpaceshipSystem,
-    spaceship_collision_handler::SpaceshipCollisionSystem,
+    enemy_destroyed::EnemyDestroyedSystem, enemy_hit::EnemyHitSystem, enemy_spawn::SpawnerSystem,
+    gamemaster::GameMasterSystem, hitbox_system::HitboxSystem, item::ItemSystem,
+    manualblaster_system::ManualBlasterSystem, planets::PlanetsSystem, player_hit::PlayerHitSystem,
+    spaceship::SpaceshipSystem, spaceship_collision_handler::SpaceshipCollisionSystem,
```

Comment:
nit: it would be nice if we could organize this to be more readable. Could we put each export on a newline?

```suggestion
    enemy_destroyed::EnemyDestroyedSystem, 
    enemy_hit::EnemyHitSystem, 
    enemy_spawn::SpawnerSystem,
    gamemaster::GameMasterSystem, 
    hitbox_system::HitboxSystem, 
    item::ItemSystem,
    manualblaster_system::ManualBlasterSystem, 
    planets::PlanetsSystem, 
    player_hit::PlayerHitSystem,
    spaceship::SpaceshipSystem, 
    spaceship_collision_handler::SpaceshipCollisionSystem,
```

Reply:
I agree with you, but I think this is rustfmt's doing. I think there is a way to create a rustfmt config file, but I think for now I'll push this off to a later issue.

Reply:
Sounds good. Thanks for filing an issue for this!